src/main/java/com/miraisolutions/xlconnect/WithJNI.java

```diff
@@ -0,0 +1,8 @@
+package com.miraisolutions.xlconnect;
+
+public interface WithJNI {
```

Comment:
There's currently only a single use with `BooleanWithAttributes`, so maybe this isn't even needed?

Reply:
Thanks! I forgot to clean this up, I was using it in a previous attempt.